Dockerfile

```
@@ -0,0 +1,21 @@
+FROM php:7.2-fpm
+
+# Install PHP extensions
+RUN docker-php-ext-install pdo_mysql && docker-php-ext-install calendar
+
+# Install Composer
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+
+# Install Git
+RUN apt-get update && apt-get install -y git
+
+# Install Node.js
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - && apt-get install -y nodejs
```

```suggestion
SHELL ["/bin/bash", "-o", "pipefail", "-c"]
RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - \
  && apt-get install -y nodejs \
  && rm -rf /var/lib/apt/lists/*
```

You want to change the shell options to fail if anything in the pipe fails. Otherwise the image could build without having internet access and be totally broken.